Userland/Libraries/LibCore/System.cpp

```diff
@@ -160,6 +150,22 @@ ErrorOr<int> accept4(int sockfd, sockaddr* address, socklen_t* address_length, i
 }
 #endif
 
+ErrorOr<void> pledge(StringView promises, StringView execpromises)
```

Comment:
Why move it around?!

Reply:
Agreed. Please don't arbitrarily move functions that already exist around the file please ;)

Reply:
It's moved from inside an `#ifdef` to outside.